student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, classroom)
```

Comment:
- [ ] Kindly include `parent_permission` in the parameter of the `initialize` method. This will make it possible to decide if an instance of a `Student` class is given permission to have access to certain services. 👍